api/src/main/java/org/openmrs/person/PersonMergeLogData.java

```diff
@@ -38,9 +38,9 @@ public class PersonMergeLogData {
 	private List<String> movedEncounters;
```

Comment:
Unfortunately this is a public API, and we really don't know how it is being used.  I think we should just deprecate the existing `createdPrograms` methods and add the new `movedPrograms` methods, rather than rename/replace them.  We can then remove them in some future release.

Reply:
Fair enough, will do.

Reply:
Yeah, I was kind of evaluating this as something added in 2.7, where removing a seldom-used API is kind of a non-issue (we don't use it anywhere else); however, for backporting to 2.6, we would need to retain (and deprecate) the "created programs" stuff.

FWIW, I couldn't find any uses of this class outside of forks of OpenMRS-Core, so it seems very unlikely to be an issue.